packages/system/gpu-operator/Makefile

```diff
@@ -0,0 +1,13 @@
+CILIUM_TAG=$(shell awk '$$1 == "version:" {print $$2}' charts/cilium/Chart.yaml)
```

Comment:
`CILIUM_TAG` here seems to be a wrong copy-paste.

Reply:
Thank you!

Reply:
fixed